### ISSUE TITLE
Replace library does replace for symbols, but not for styles

### ISSUE THREAD

**Issue by oodesign** [Ashung/Automate-Sketch#287]
Hi!

I'm trying to use "Libraries > Replace Library", and it seems to work fine with symbols, but not with styles.

Case is:

1. Have a design file ("DesignFile"), that uses a bunch of symbols, layer styles, and text styles from a library ("Library").
2. Create a duplicate of "Library" (in this case, because we want to create a new version of the library, and keep the old one). Let's call this one "Library_2", and define it as a linked library.
3. Create a duplicate of the "DesignFile". We'll call that one "DesignFile_2".
4. Open "DesignFile_2", and run "Automate > Libraries > Replace Library".
5. Choose to replace "Library" for the new one, "Library_2".
6. All symbols seem to be changed (I'm checking that with script), but some layer & text styles don't (don't know if all of them, or just a part), and still have the reference to the old "Library".

Is it an issue? Is there any workaround?

I'm using Sketch 68.2, on macOS Catalina 10.15.6, and Automate Sketch 66.1.20200613.

Love this plugin. I'm a huge fan 🥰.

**Comment by Ashung** [comment]
Replace library is only for the 2 libraries have different document ID, but same symbol/style ID. you duplicate library file in Finder, and use "Library - Fix Library ID Conflict" to change the document ID for new library.

And if you make a new library document from Sketch, use "Symbol - Find and Replace Symbol" and "Style - Find and Replace Layer/Text Style".

For symbols, you can make all library symbols local ("Library - Change Library Symbols to Local Symbol"), then replace them to library symbol with same name (""Library - Change Local Symbols to Library Symbol"").

**Comment by oodesign** [comment]
Hi @Ashung ,

It took a while but we found the issue (and it's on our side). 
For some reason the secondary file got corrupt (in the foreignLayerStyles collection), and it was causing several issues down the path.

We created the file again (and checked everything was OK) and now Replace Library works like a charm.

Thanks a lot, and sorry for having you on top of this issue, being our fault. 

Thank you! 😊
